### ISSUE TITLE
Pick is not defined in abacus-pricing-config

### ISSUE THREAD

**Issue by rajkiranrbala** [cloudfoundry-incubator/cf-abacus#288]
Found that pick is not defined in abcaus-pricing-config

```sh
/Users/rajkiranrbala/Projects/OpenSource/cf-abacus/node_modules/abacus-pricing-config/lib/index.js:96
        return unlock(cb(extend({}, pick(val, ['statusCode', 'body']), { heade
                                    ^
ReferenceError: pick is not defined
    at /Users/rajkiranrbala/Projects/OpenSource/cf-abacus/node_modules/abacus-pricing-config/lib/index.js:96:37
    at /Users/rajkiranrbala/Projects/OpenSource/cf-abacus/node_modules/abacus-retry/lib/index.js:86:9
    at callback (/Users/rajkiranrbala/Projects/OpenSource/cf-abacus/node_modules/abacus-breaker/lib/index.js:70:125)
    at /Users/rajkiranrbala/Projects/OpenSource/cf-abacus/node_modules/abacus-breaker/lib/index.js:101:14
    at /Users/rajkiranrbala/Projects/OpenSource/cf-abacus/node_modules/abacus-batch/lib/index.js:85:30
    at _.map._.collect (/Users/rajkiranrbala/Projects/OpenSource/cf-abacus/node_modules/underscore/underscore.js:172:24)
    at /Users/rajkiranrbala/Projects/OpenSource/cf-abacus/node_modules/abacus-batch/lib/index.js:82:14
    at /Users/rajkiranrbala/Projects/OpenSource/cf-abacus/node_modules/abacus-request/lib/index.js:300:11
    at /Users/rajkiranrbala/Projects/OpenSource/cf-abacus/node_modules/abacus-transform/lib/index.js:97:16
    at /Users/rajkiranrbala/Projects/OpenSource/cf-abacus/node_modules/abacus-request/lib/index.js:290:18
    at /Users/rajkiranrbala/Projects/OpenSource/cf-abacus/node_modules/abacus-transform/lib/index.js:83:5
    at _.map._.collect (/Users/rajkiranrbala/Projects/OpenSource/cf-abacus/node_modules/underscore/underscore.js:172:24)
    at Function.cbmap (/Users/rajkiranrbala/Projects/OpenSource/cf-abacus/node_modules/abacus-transform/lib/index.js:81:3)
    at /Users/rajkiranrbala/Projects/OpenSource/cf-abacus/node_modules/abacus-request/lib/index.js:283:19
    at /Users/rajkiranrbala/Projects/OpenSource/cf-abacus/node_modules/abacus-transform/lib/index.js:97:16
    at Object.callback (/Users/rajkiranrbala/Projects/OpenSource/cf-abacus/node_modules/abacus-request/lib/index.js:265:13)
    at /Users/rajkiranrbala/Projects/OpenSource/cf-abacus/node_modules/abacus-request/lib/index.js:172:14
    at Request._callback (/Users/rajkiranrbala/Projects/OpenSource/cf-abacus/node_modules/abacus-request/lib/index.js:154:14)
    at Request.self.callback (/Users/rajkiranrbala/Projects/OpenSource/cf-abacus/node_modules/request/request.js:199:22)
    at Request.emit (events.js:110:17)
    at Request.<anonymous> (/Users/rajkiranrbala/Projects/OpenSource/cf-abacus/node_modules/request/request.js:1036:10)
    at Request.emit (events.js:129:20)
    at IncomingMessage.<anonymous> (/Users/rajkiranrbala/Projects/OpenSource/cf-abacus/node_modules/request/request.js:963:12)
    at IncomingMessage.emit (events.js:129:20)
    at _stream_readable.js:908:16
    at process._tickCallback (node.js:355:11)
```

**Comment by betafood** [comment]
As an additional note, since I believe all the tests work off of ```test-pricing-basic``` or ```test-pricing-standard```, this was not caught in them.

**Comment by KRuelY** [comment]
Thanks for catching that! Should be fixed with 26995a8afb4ae7d371348c5b3fa23c9e8757ee5d
